### PR TITLE
create view admin CRUD product

### PR DESCRIPTION
**Pull request by NgocAnhDT**
## Related Tickets:
- [#46205](https://edu-redmine.sun-asterisk.vn/projects/hn_oe49-shoe-store/issues/46205)

## WHAT this PR does?
- Create view admin CRUD product

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by NgocAnhDT** [comment]
ready

File: resources/views/products/create.blade.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
+@extends('layouts.appAdmin')
+
+@section('title')
+    Create Product
```
Comment (manhtd-1083) [inline resources/views/products/create.blade.php:4]: localization

File: resources/views/products/edit.blade.php [hunk 2]
```diff
@@ -0,0 +1,71 @@
+@extends('layouts.appAdmin')
+
+@section('title')
+    Edit Product
```
Comment (manhtd-1083) [inline resources/views/products/edit.blade.php:4]: localization

**Comment by manhtd-1083** [comment]
CI fail

**Comment by NgocAnhDT** [comment]
ready

**Pull request merged**